src/main/java/org/skife/jdbi/v2/PreparedBatch.java

```diff
@@ -109,7 +110,22 @@ public PreparedBatch define(final Map<String, ? extends Object> values)
                         getContext(),
                         getContainerMapperRegistry());
             }
-        });
+        }, null);
+
+    }
+
+    @SuppressWarnings("unchecked")
+    public <GeneratedKeyType> GeneratedKeys<GeneratedKeyType> executeAndGenerateKeys(final ResultSetMapper<GeneratedKeyType> mapper,
+                                                                                     String[] columnNames) {
```

Comment:
I think the varargs declaration is quite fine. It provides a more terse API and works the same way internally as the explicit array declaration.